pkg/sentry/socket/netstack/netstack.go

```diff
@@ -3111,6 +3112,12 @@ func interfaceIoctl(ctx context.Context, io usermem.IO, arg int, ifr *linux.IFRe
 		// Gets the MTU of the device.
 		hostarch.ByteOrder.PutUint32(ifr.Data[:4], iface.MTU)
 
+	case linux.SIOCSIFMTU:
+		MTU := hostarch.ByteOrder.Uint32(ifr.Data[:4])
```

Comment:
nit: BTW, code convention wise I think this variable name should be 'mtu'

Reply:
Fixed :)